trash/renderer.cpp

```diff
@@ -17,14 +17,14 @@ namespace gui {
 
 namespace render {
 
-MainRenderer::MainRenderer() {
+MainRenderer::MainRenderer(ShaderHandeler shader_handler) {
```

Comment:
```suggestion
MainRenderer::MainRenderer(ShaderHandeler shader_handler) {
```

Reply:
> goes into trash file
> suggests changes

kinda on me I should have deleted this already.